code/modules/clothing/shoes/cluwne.dm

```diff
@@ -0,0 +1,35 @@
+/obj/item/clothing/shoes/cluwne
+	desc = "The prankster's standard-issue clowning shoes. Damn, they're huge!"
+	name = "clown shoes"
+	icon_state = "clown"
+	inhand_icon_state = "clown_shoes"
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	item_flags = DROPDEL
+	slowdown = SHOES_SLOWDOWN+1
```

Comment:
why + 1 and not a new define?

Reply:
doesn't seem to be how the define works, look at clown's shoes also, galoshes

Reply:
`SHOES_SPEED_SLIGHT` (SHOES_SLOWDOWN - 1) exists, why not have the inverse?

Reply:
SHOES SPEED SLIGHT is not used anywhere btw